backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/entity/tire/TireProperties.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.muni.fi.pa165.pneuservis.backend.entity.tire;
+
+import cz.muni.fi.pa165.pneuservis.backend.enums.SeasonEnum;
+import cz.muni.fi.pa165.pneuservis.backend.enums.TireVehicleTypeEnum;
+import cz.muni.fi.pa165.pneuservis.backend.enums.SpeedClassEnum;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+/***
+ * @author Jakub Palenik, 422453@mail.muni.cz
+ */
+@Entity
+public class TireProperties {

```

Comment:
This shouldn't be separate. These fields should be inside of Tire class.

Reply:
This should be separate, otherwise there would be redundant data in table Tire.  One tire property  has multiple different tires.

Reply:
It's higly improbable that there'd be multiple tires from different manufacturers with the exact same properties. I wouldn't consider it redundancy if some subset of the fields repeats in multiple tires.

Reply:
On the contrary it is higly probable. Because to maximize your profit as a company you want to sell pneus for most used tire sizes.